.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules
 /dist
-.idea
 npm-debug.log
+package-lock.json
```

Comment:
I have read that package-lock.json is intended to be committed to version control.

Is there a reason that we do not want it here?

Reply:
package-lock.json is intended for version control, it was my mistake to remove it. I'll correct it.